routing/restrictions_serialization.hpp

```diff
@@ -35,8 +37,12 @@ struct Restriction
   /// * driving along the restriction is prohibited (No)
   enum class Type
   {
-    No,    // Going according such restriction is prohibited.
-    Only,  // Only going according such restriction is permitted
+    No,        // Going according such restriction is prohibited.
+    Only,      // Only going according such restriction is permitted.
+
+    NoUTurn,   // Making U-turn from a feature to the same one according
+               // to such restriction is prohibited.
+    OnlyUTurn
```

Comment:
And a comment for it?

Reply:
`Only U-turn is permitted at the feature.`